lib/fabrik/DirectorManager.js

```diff
@@ -59,14 +58,14 @@ class DirectorManager extends BaseManager {
   }
 
   get networkName() {
-    return this.subnet || infrastructure.segmentation.network_name || 'default';
+    return this.subnet || this.director.config.infrastructure.segmentation.network_name || 'default';
```

Comment:
comments similar to [above](https://github.com/SAP/service-fabrik-broker/pull/7/files#r133368904)